integration/raft/channel_participation_test.go

```diff
@@ -255,6 +257,148 @@ var _ = Describe("ChannelParticipation", func() {
 			}
 		})
 	})
+
+	Describe("three node etcdraft network with a system channel", func() {
+		startOrderer := func(o *nwo.Orderer) {
+			ordererRunner := network.OrdererRunner(o)
+			ordererProcess := ifrit.Invoke(ordererRunner)
+			Eventually(ordererProcess.Ready(), network.EventuallyTimeout).Should(BeClosed())
+			ordererProcesses = append(ordererProcesses, ordererProcess)
+			ordererRunners = append(ordererRunners, ordererRunner)
+		}
+
+		restartOrderer := func(o *nwo.Orderer, index int) {
+			ordererProcesses[index].Signal(syscall.SIGKILL)
+			Eventually(ordererProcesses[index].Wait(), network.EventuallyTimeout).Should(Receive(MatchError("exit status 137")))
+			ordererRunner := network.OrdererRunner(o)
+			ordererProcess := ifrit.Invoke(ordererRunner)
+			Eventually(ordererProcess.Ready(), network.EventuallyTimeout).Should(BeClosed())
+			ordererProcesses[index] = ordererProcess
+			ordererRunners[index] = ordererRunner
+		}
+
+		BeforeEach(func() {
+			network = nwo.New(nwo.MultiNodeEtcdRaft(), testDir, client, StartPort(), components)
+			network.GenerateConfigTree()
+			network.Bootstrap()
+		})
+
+		It("joins channels using the legacy channel creation mechanism and then removes the system channel to transition to the channel participation API", func() {
+			orderer1 := network.Orderer("orderer1")
+			orderer2 := network.Orderer("orderer2")
+			orderer3 := network.Orderer("orderer3")
+			orderers := []*nwo.Orderer{orderer1, orderer2, orderer3}
+			peer := network.Peer("Org1", "peer0")
+			for _, o := range orderers {
+				startOrderer(o)
+			}
+
+			findLeader(ordererRunners)
+
+			By("creating an application channel using system channel")
+			network.CreateChannel("testchannel", orderer1, peer)
+
+			By("broadcasting envelopes to each orderer")
+			for _, o := range orderers {
+				env := CreateBroadcastEnvelope(network, peer, "testchannel", []byte("hello"))
+				resp, err := ordererclient.Broadcast(network, o, env)
+				Expect(err).NotTo(HaveOccurred())
+				Expect(resp.Status).To(Equal(common.Status_SUCCESS))
+			}
+
+			By("enabling the channel participation API on each orderer")
+			network.Consensus.ChannelParticipationEnabled = true
```

Comment:
Done. Set bootstrap method to none. 

For the second part, I'll create a JIRA to investigate the start sequence for the orderer after removing the system channel. The immediate thing that jumps out is that it will log "Not bootstrapping the system channel because of existing channels" at the info level if the bootstrap method hasn't been updated from file.

Reply:
Created [FAB-18230](https://jira.hyperledger.org/browse/FAB-18230)